test/context.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
this would be way more legible on multiple lines... think FP (functional programming).